### DISCUSSION TITLE
Anti-claymining

### DISCUSSION THREAD

**Discussion by agoniaprivat** [PurpurMC/Purpur#475] [__Dismissed__] [answered]
**Is your feature request related to a problem?**
Please add a way to remove the Clay Mining technique. It's destroying the survival feeling.
https://www.youtube.com/watch?v=Ufkx7j67Unk

**Describe the solution you'd like.**
Change the diamond generation somehow.

**Describe alternatives you've considered.**
Removing diamonds from the map.

**Comment by BillyGalbreath** [comment, marked as answer]
Already did. See the feature seed options in purpur.yml ;)
